wallet/src/account/currency_grouper/mod.rs

```diff
@@ -128,6 +130,36 @@ pub fn group_outputs_with_issuance_fee<T, Grouped: Clone>(
     Ok(tokens_grouped)
 }
 
+fn output_spendable_value(output: &TxOutput) -> Result<(Currency, Amount), UtxoSelectorError> {
+    let output_value = match output {
+        TxOutput::Transfer(v, _) | TxOutput::LockThenTransfer(v, _, _) => v.clone(),
+        TxOutput::IssueNft(token_id, _, _) => {
+            OutputValue::TokenV1(*token_id, Amount::from_atoms(1))
```

Comment:
Is this a mistake?

Reply:
no, you can spend IssueNft as input, but it could be clearer if it just returns it directly as (Currencty::Token(token_id), amount(1)) instead of making an OutputValue and then extracting it from there again.